yarn.lock

```diff
@@ -2,35 +2,29 @@
 # yarn lockfile v1
```

Comment:
It seems the lockfile was almost completely rebuilt, though I do not see any `package.json` changes.
Are these changes necessary? If not, could you revert this file?